_posts/2019-05-06-live-view-with-presence.md

```diff
@@ -0,0 +1,581 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: til
```

Comment:
We should figure out what category this really is. @burden sent me some sneak peeks at fancy support for Categories and Tags.

Reply:
Yea would love to start getting this right! Open to any suggestions from the "sneak peak"

Reply:
I'll update the category and add tags after this PR goes out! https://github.com/elixirschool/elixirschool/pull/1837